drivers/wd/setup.py

```diff
@@ -0,0 +1,20 @@
+from setuptools import setup, find_packages
+
+setup(
+    name="onitu-webdav",
+    version="0.1",
+    url="http://onitu.github.io",
+    description="Store files on a remote machine using webdav through onitu.",
+    license="MIT",
+    packages=find_packages(),
+    install_requires=["pycurl", "lxml", "webdavclient"],
+    package_data={'': ['manifest.json']},
+    entry_points={
+        'onitu.drivers': [
+            'wd = onitu_webdav'
```

Comment:
Well, here it's the name for Onitu, this has nothing to do with Python packages. It's namespaced by 'onitu.drivers' so that's fine. Your package name is `onitu_webdav`, so no collision here.